data/crac-creation/crac-creator-csa-profiles/src/main/java/com/farao_community/farao/data/crac_creation/creator/csa_profile/crac_creator/cnec/CsaProfileFlowCnecCreationContext.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.farao_community.farao.data.crac_creation.creator.csa_profile.crac_creator.cnec;
+
+import com.farao_community.farao.data.crac_creation.creator.api.ElementaryCreationContext;
+import com.farao_community.farao.data.crac_creation.creator.api.ImportStatus;
+
+/**
+ * @author Jean-Pierre Arnould {@literal <jean-pierre.arnould at rte-france.com>}
+ */
+public final class CsaProfileFlowCnecCreationContext implements ElementaryCreationContext {
```

Comment:
I don't know how you intend to implement voltage cnecs'import but maybe you could stay generic in class naming to be able to reuse class for voltage cnecs : CsaProfileCnecCreationContext

Reply:
ok, I hesitated on naming... done